internal/manifest/build.go

```diff
@@ -15,16 +16,18 @@ import (
 type BuildPipeline struct {
 	BasePipeline
 
+	runner       runner.Runner
```

Comment:
This is interesting since it is like the kernel version a transient property, i.e. it depends on the specific package set instead of some configuration.